azure-pipelines.yml

```diff
@@ -20,6 +20,12 @@ variables:
   core-package: $(core-version)$(core-release)
 
 steps:
+- task: UseDotNet@2
```

Comment:
We need to install .NET 6 here as the build server `windows-latest` image doesn't come with .NET 6 SDK installed, however, it does come with .NET Core 3.1 and .NET 5.